daisy_workflows/image_build/debian/debian_9.wf.json

```diff
@@ -17,11 +24,36 @@
           "bootstrap_vz_manifest": "official/gce/stretch.yml",
           "bootstrap_vz_version": "${bootstrap_vz_version}",
           "builder_source_image": "projects/compute-image-tools/global/images/family/debian-9-worker",
-          "google_cloud_repo": "${google_cloud_repo}",
-          "image_dest": "${image_dest}"
+          "google_cloud_repo": "${google_cloud_repo}"
         }
       }
+    },
+    "copy-gcs-object": {
+      "CopyGCSObjects": [
+        {
+          "Source": "${OUTSPATH}/root.tar.gz",
+          "Destination": "${OUTSPATH}/root.tar.gz"
+        }
+      ]
+    },
+    "create-image": {
+      "CreateImages": [
+        {
+          "Name": "debian-9-v${build_date}",
+          "RawDisk": {
+            "Source": "${OUTSPATH}/root.tar.gz"
+          },
+          "Description": "Debian, Debian, 9, x86_64 built on ${build_date}",
```

Comment:
Please use the name and descriptions we are using for the regular publication and add the relevant licenses and Guest OS features.

Reply:
I will refer publish.json `description, `licenses` and `guestOSfeatures` fields. For license, I will use prod licences. In description, use ${build_date} replace {{time}}.